### PR TITLE
Event details

### PR DESCRIPTION
**Pull request by oankit**
Made style changes, added a script to edit database fields, and created the details page. (The admin credentials file was removed because I am not allowed to push a private key on the repo; PM to get the JSON file if needed).

### CODE REVIEW

File: code/drocsid/src/firebase/editevents.js
Comment (ammaarkhan) [inline code/drocsid/src/firebase/editevents.js:1]: what is happening over here?? why is there this constant for eventdetails? shouldn't this be stored in the backend?

**Review by ammaarkhan — COMMENTED** [review]
need some clarification on editevents.js

File: code/drocsid/src/firebase/editevents.js
Comment (oankit) [reply code/drocsid/src/firebase/editevents.js:1]: eventdetails fetched the required info from firestone using the eventid. and then displays it on he page.

Comment (oankit) [reply code/drocsid/src/firebase/editevents.js:1]: editevents.js was used to add mock data to the database, it's a script. I create a firebase admin SDK to edit the database but I couldn't include it in my commit due to github restrictions.

Comment (ammaarkhan) [reply code/drocsid/src/firebase/editevents.js:1]: oh okay got it, thanks for the clarification - all looks good!

**Review by ammaarkhan — APPROVED** [review]

**Review by MuradSh — APPROVED** [review]

**Pull request merged**